content/applications/sales/sales/amazon_connector/manage.rst

```diff
@@ -53,6 +53,29 @@ turn, notify the customer that the order (or a part of it) is on its way.
 .. seealso::
    - :doc:`../../../inventory_and_mrp/inventory/shipping/setup/third_party_shipper`
 
+.. _manage/manage_delivery_errors:
+
+Manage errors when synchronizing the deliveries
+-----------------------------------------------
+
+When Odoo verifies that all the information sent to Amazon are correctly processed, it can happen
+that Amazon failed to do so. In that case, Odoo send an email listing all the shipments that failed
```

Comment:
Editing this section for clarity and wordiness. I assume that Odoo sends an email to the Odoo database user in this case, but please correct me if I'm wrong.
```suggestion
Sometimes, Amazon can fail to correctly process all the information sent by Odoo. In this case, Odoo sends an email to the Odoo user listing all the shipments that failed
```